nucliadb/nucliadb/search/api/v1/suggest.py

```diff
@@ -179,4 +183,14 @@ async def suggest_knowledgebox(
     response.status_code = 206 if incomplete_results else 200
     if debug:
         search_results.shards = queried_shards
+
+    if audit is not None:
+        await audit.suggest(
+            kbid,
+            x_nucliadb_user,
+            x_ndb_client.to_proto(),
+            x_forwarded_for,
+            timeit - time(),
```

Comment:
this should be
```python
time() - timeit
```

Reply:
wow good catch. There's at least one other place where this is wrong, not from this PR; i'll fix it

Reply:
Fixed